R/praise.R

```diff
@@ -62,15 +6,21 @@
 #'
 #' @examples
 #' replicate(5, glue::glue("Random praise: {random_praise()}"))
-#' replicate(5, glue::glue("Random encouragement: {random_encourage()}"))
+#' replicate(5, glue::glue("Random encouragement: {random_encouragement()}"))
 #' @export
 #' @describeIn praise Returns a random praise message
 random_praise <- function() {
-  sample(.praise, 1)
+  sub("^Correct! ", "", learnr::random_praise())
 }
 
 #' @export
-#' @describeIn praise Returns a random encouragement message
+#' @describeIn praise Returns a random encouragement message, included for 
```

Comment:
Can we move `random_encourage()` definition and docs to https://github.com/rstudio-education/gradethis/blob/master/R/deprecated.R ?

This doesn't expose `random_encourage()` when looking at the docs and makes it obvious the function should not be used.